test/suite/test_live_restore04.py

```diff
@@ -0,0 +1,124 @@
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import filecmp, os, glob, wiredtiger, wttest
+from wtdataset import SimpleDataSet
+from wtscenario import make_scenarios
+from helper import copy_wiredtiger_home
+from suite_subprocess import suite_subprocess
+
+
+# test_live_restore04.py
+# Test using the wt utility with live restore.
+@wttest.skip_for_hook("tiered", "using multiple WT homes")
+class test_live_restore04(wttest.WiredTigerTestCase, suite_subprocess):
+    format_values = [
+        ('column', dict(key_format='r', value_format='S')),
+        ('row_integer', dict(key_format='i', value_format='S')),
+    ]
+    scenarios = make_scenarios(format_values)
+    nrows = 10000
+    ntables = 3
+    conn_config = 'log=(enabled)'
+
+    def get_stat(self, statistic):
+        stat_cursor = self.session.open_cursor("statistics:")
+        val = stat_cursor[statistic][2]
+        stat_cursor.close()
+        return val
+
+    def test_live_restore04(self):
+        # FIXME-WT-14051: Live restore is not supported on Windows.
+        if os.name == 'nt':
+            self.skipTest('Unix specific test skipped on Windows')
+
+        # Create a folder to save the wt utility output.
+        util_out_path = 'UTIL'
+        os.mkdir(util_out_path)
+
+        uris = []
+        for i in range(self.ntables):
+            uri = f'file:collection-{i}'
+            uris.append(uri)
+            ds = SimpleDataSet(self, uri, self.nrows, key_format=self.key_format,
+                               value_format=self.value_format)
+            ds.populate()
+
+        # Dump file data for later comparison.
+        for i in range(self.ntables):
+            dump_out = os.path.join(util_out_path, f'{uris[i]}.out')
+            self.runWt(['dump', '-x', uris[i]], outfilename=dump_out)
+
+        # Close the default connection.
+        self.close_conn()
+
+        copy_wiredtiger_home(self, '.', "SOURCE")
+
+        # Remove everything but SOURCE / stderr / stdout / util output folder.
+        for f in glob.glob("*"):
+            if not f == "SOURCE" and not f == "UTIL" and not f == "stderr.txt" and not f == "stdout.txt":
+                os.remove(f)
+
+        # Open a live restore connection with no background migration threads to leave it in an
+        # unfinished state.
+        self.open_conn(config="log=(enabled),statistics=(all),live_restore=(enabled=true,path=\"SOURCE\",threads_max=0)")
+        self.close_conn()
+
+        # Check that opening the wt utility without a proper live restore path gives an error.
+        self.runWt(['dump', '-x', uris[0]],
+                   outfilename=f'{uris[0]}.lr.out',
+                   errfilename='wterr.txt',
+                   reopensession=False,
+                   failure=True)
```

Comment:
```suggestion
        lr_dump_out = os.path.join(util_out_path, f'{uris[i]}-error.lr.out')
        self.runWt(['dump', '-x', uris[0]],
                   outfilename=lr_dump_out,
                   errfilename='wterr.txt',
                   reopensession=False,
                   failure=True)
```
Just to chuck it in the UTIL directory for neatness.

Reply:
03ac827f9